webapp/src/ts/modules/contacts/contacts-content.component.ts

```diff
@@ -46,20 +47,33 @@ export class ContactsContentComponent implements OnInit, OnDestroy {
     this.subscribeToChanges();
     this.setReportsTimeWindowMonths(3);
     this.setTasksTimeWindowWeeks(1);
+    this.getUserFacility();
   }
 
   ngOnDestroy() {
     this.subscription.unsubscribe();
   }
 
+  getUserFacility() {
+    const facilitySubscription = combineLatest(
+      this.store.select(Selectors.getUserFacilityId))
+      .subscribe(([userFacilityId]) => {
+        this.userFacilityId = userFacilityId;
+        if (this.userFacilityId && !this.route.snapshot.params.id) {
+          this.contactsActions.selectContact(this.userFacilityId);
```

Comment:
I think you're going to need to add an additional condition here, to check whether we are mobile / showing content because:
1. Right now, on mobile view, navigating to the contacts tab loads the detail of the facility instead of the list.
2. On mobile, we don't want to load the user's facility for performance reasons (user doesn't see it anyway?). 

Please update the unit tests to reflect this.